src/OctoshiftCLI.Tests/ado2gh/Commands/AbortMigration/AbortMigrationCommandTests.cs

```diff
@@ -0,0 +1,21 @@
+using FluentAssertions;
+using OctoshiftCLI.AdoToGithub.Commands.WaitForMigration;
```

Comment:
this is redundant
```suggestion
```

Reply:
Shoot tests are broken, are you sure we dont need this ?

Reply:
Oh the reason is the namespaces are wrong in `AbortMigrationCommand`s, will add new code suggestions.